broker-daemon/utils/nano-to-datetime.js

```diff
@@ -0,0 +1,17 @@
+const nano = require('nano-seconds')
+
+/**
+ * Converts nanoseconds string to ISO8601 datetime
+ *
+ * @param {String} nanoseconds
+ * @return {String} ISO8601 string
+ */
+function nanoToDatetime (nanoseconds) {
+  const first = nanoseconds.substring(0, nanoseconds.length - 9)
+  const second = nanoseconds.substring(nanoseconds.length - 8, nanoseconds.length)
```

Comment:
This looks like it needs to be `nanoseconds.length - 9`, or else we will be missing the first digit